go/src/koding/klient/machine/machinegroup/syncs/syncs.go

```diff
@@ -141,7 +124,23 @@ func (s *Syncs) worker() {
 
 // Add starts synchronization between remote and local directories. It creates
 // all necessary cache files if they are not present.
-func (s *Syncs) Add(mountID mount.ID, m mount.Mount, dynClient client.DynamicClientFunc) error {
+func (s *Syncs) Add(mountID mount.ID, m mount.Mount,
+	nb notify.Builder, sb msync.Builder,
+	dynSSH msync.DynamicSSHFunc, dynClient client.DynamicClientFunc) error {
```

Comment:
Changed in #10723 [here](https://github.com/koding/koding/pull/10723/files#diff-43fb569605c334541ac3541ce20a9ea0R126)